saleor/plugins/base_plugin.py

```diff
@@ -22,6 +21,19 @@
 PluginConfigurationType = List[dict]
 
 
+class ConfigurationTypeField:
```

Comment:
I would consider extracting this class to separate file. What do you think?

Reply:
Before, this class was in __init__ file, I can move it there again, that's ok?